ebuild-writing/misc-files/metadata/text.xml

```diff
@@ -140,6 +141,14 @@ metadata.xml:
     tag may appear at most once.
   </ti>
 </tr>
+<tr>
+  <ti><c>&lt;stabilize-allarches&gt;</c></ti>
+  <ti>
+    Indicates an architecture-independent package. An ebuild can be marked
+    stable for all relevant architectures, when it has been tested on only one
+    architecture. This tag must have empty content.
```

Comment:
Maybe it's worth adding that this applies only if the same version has `~arch` keywords already. I think a lot of devs are missing that.

Reply:
The word "relevant" was supposed to convey that. :) Any suggestion for a more explicit wording? Keep in mind that the goal here is to describe the metadata, but not necessarily the exact workflow of arch testers.

Reply:
Nah, it's good enough as is.